apps/arweave/src/ar_serialize.erl

```diff
@@ -2120,6 +2120,8 @@ encode_packing(spora_2_5, _Strict) ->
 encode_packing(unpacked, _Strict) ->
 	"unpacked".
 
+decode_packing(<<"any">>, _Error) ->
```

Comment:
I think the `any` case was omitted before because `any` isn't a valid packing format. You'll see above under `encode_packing` that the `any` case is only handled when `Strict` is `false`. The idea is that we only want to handle encoding or decoding `any` if we're explicitly in a code block that doesn't need a valid packing format. If however we're in a code block that expects a valid packing format, `any` or `<<"any">>` should cause an error.

Im not sure what the implication are of supportin `<<"any">>` in all decode paths. Could be fine, but worth double checking that all of the code paths that call decode_packing either explicitly allow for `any` or have another chekc that validates that the packing format is correct.